unit_test.sh

```diff
@@ -10,47 +10,11 @@ timestamp=`date +%Y-%m-%d_%H-%M-%S`
 mkdir -p test_results/${timestamp}
 mkdir -p test_results/latest
 
-echo "Setting up test DB (bcda_test)..."
```

Comment:
I'm worried, that if we eliminate this setup script, it will be difficult for a developer to re-configure/maintain the state of the unit test database.  I wonder if we should create a unit test migration script, which can be used as a starting point for the image...  that way, if anyone needs to change it, they can start with this script.
Ideally, we'd move to a mocking strategy where we mock the DB.  But in the meantime, this will be a good improvement.

Reply:
Let me update the comment in the README. The `dump.pgdata` and `01-restore.sh` files are intended as the starting point of the image. The pg_restore command in the script encompasses all of the `usql` and `import-synthetic-cclf-package` commands.

Reply:
Yup, I totally get that the packaged `pgdata` file encompasses all of the `usql` and `import-synthetic-cclf-package` commands.  I'm just a little concerned from a historical perspective... future maintainers might not know how to make adjustments to the data using the legacy mechanism.  Using the `pgrestore` makes this process much faster from a CI perspective, but it might cause challenges from a DB-maintenance perspective.  Is there any possible way, for maintenance purposes, to add the legacy mechanism for creating the base DB image?  Maybe we can add the legacy script using `usql` and `import-synthetic-cclf-package` as a giant comment section in `01-restore.sh`?

Reply:
That makes sense. I'll go ahead and update the restore script to capture the arguments.